GPSTest/src/main/java/com/chesapeaketechnology/gnssmonkey/AGpsMonkeyActivity.java

```diff
@@ -37,6 +38,7 @@
     protected static final String TAG = "GPSMonkey.Activity";
     private static final String PREF_BATTERY_OPT_IGNORE = "nvroptbat";
     private static final int PERM_REQUEST_CODE = 1;
+    private final Intent serviceIntent = new Intent(this, GpsMonkeyService.class);
```

Comment:
I get a NPE if the Intent is initialized here.  This works for me:
```suggestion
    private Intent serviceIntent;
```